src/test/java/karen/tasks/TaskListTest.java

```diff
@@ -0,0 +1,32 @@
+package karen.tasks;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertArrayEquals;
+
+public class TaskListTest {
+
+    @Test
+    public void toTaskStrings_emptyTaskList_emptyStringArray() {
```

Comment:
Are test methods supposed to be in snake case? If so, is it legal to mix between snake and camel cases?